content/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -12,110 +12,97 @@ description: >-
+Click **SSO** in the sidebar to review and [remove SSO identity links](/cloud-docs/users-teams-organizations/single-sign-on/linking-user-account#remove-sso-identity-link) associated with your account.
 
-API tokens are necessary for:
+You have an SSO identity for every SSO-enabled Terraform Cloud organization. Terraform Cloud links each SSO identity to a single Terraform Cloud user account. This link determines which account you can use to access each organization.
 
-- Authenticating with the [Terraform Cloud API](/cloud-docs/api-docs). API calls require an `Authorization: Bearer <TOKEN>` HTTP header.
-- Authenticating with the [Terraform `remote` backend](/language/settings/backends/remote), which requires a token in the CLI config file or in the backend configuration.
-- Using [private modules](/cloud-docs/registry/using) in command-line Terraform runs on local machines requires [a token in the CLI config file](/cloud-docs/registry/using#authentication).
+### Tokens
 
-Terraform Cloud has three kinds of API tokens: user, team, and organization. For more information about team and organization tokens, see [API Tokens](/cloud-docs/users-teams-organizations/api-tokens).
+Click **Tokens** in the sidebar to create, manage, and revoke API tokens. Terraform Cloud has three kinds of API tokens: user, team, and organization. Users can be members of multiple organizations, so user tokens work with any organization where the associated user is a member. Refer to [API Tokens](/cloud-docs/users-teams-organizations/api-tokens) for details.
 
-Protect your tokens carefully, because they can do anything your user account can. For example, if you belong to a team with permission to read and write variables for a workspace, your API token can edit variables in that workspace. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
+API tokens are required for the following tasks:
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+- Authenticating with the [Terraform Cloud API](/cloud-docs/api-docs). API calls require an `Authorization: Bearer <TOKEN>` HTTP header.
+- Authenticating with the [Terraform Cloud CLI integration](/cli/cloud/settings) or the [`remote` backend](/language/settings/backends/remote). These require a token in the CLI config file or in the backend configuration.
+- Using [private modules](/cloud-docs/registry/using) in command-line runs on local machines. This requires [a token in the CLI config file](/cloud-docs/registry/using#authentication).
 
-Since users can be members of multiple organizations, user tokens work with any organization their user belongs to.
+Protect your tokens carefully because they can do anything your user account can. For example, if you belong to a team with permission to read and write variables for a workspace, your API token can edit variables in that workspace. Refer to [permissions](/cloud-docs/users-teams-organizations/permissions) for more details.
```

Comment:
```suggestion
Protect your tokens carefully because they contain the same permissions as your user account. For example, if you belong to a team with permission to read and write variables for a workspace, another user could use your API token to authenticate as your user account and also edit variables in that workspace. Refer to [permissions](/cloud-docs/users-teams-organizations/permissions) for more details.
```
I just want to rephrase this as another option which clicked a bit easier in my head, referring to the API token as a tool with permissions instead of it being the actor making the changes.

Reply:
this is much much better. Thank you! I phoned it in and didn't rewrite as much as I should have here. I appreciate you!